railties/lib/rails/application/configuration.rb

```diff
@@ -443,23 +443,12 @@ def colorize_logging=(val)
       #   config.session_store :my_custom_store
       def session_store(new_session_store = nil, **options)
         if new_session_store
-          if new_session_store == :active_record_store
-            begin
-              ActionDispatch::Session::ActiveRecordStore
-            rescue NameError
-              raise "`ActiveRecord::SessionStore` is extracted out of Rails into a gem. " \
-                "Please add `activerecord-session_store` to your Gemfile to use it."
-            end
-          end
-
           @session_store = new_session_store
           @session_options = options || {}
         else
           case @session_store
           when :disabled
             nil
-          when :active_record_store
-            ActionDispatch::Session::ActiveRecordStore
           when Symbol
             ActionDispatch::Session.const_get(@session_store.to_s.camelize)
```

Comment:
The idea here is to just rely on the logic in the `when Symbol` branch to resolve `:active_record_store`, yes?  Sounds good! :+1:

Reply:
Yep, I left the test to ensure it keeps working too: https://github.com/rails/rails/blob/b88de15888b2ced7cac03508e1c28845e3d30034/railties/test/application/configuration_test.rb#L1641